### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by DanMcNamee42** [OpenMAQS/.github#3] [General]
Lots of great updates on the .Net version the last few months.  Any idea on when we might see a new NuGet package?  We could use some of the updates.  Especially the screenshot repair for the newest Selenium versions.

**Comment by twalshOG** [comment]
> Lots of great updates on the .Net version the last few months.  Any idea on when we might see a new NuGet package?  We could use some of the updates.  Especially the screenshot repair for the newest Selenium versions.

New version was released last month
https://www.nuget.org/packages/OpenMAQS.Maqs/

**Comment by DanMcNamee42** [comment]
Hi Troy,

Thank you for responding.  Yes, we did get the update last month.  Thank you!  It was much needed.  Now we are waiting on the Appium release version update.  You’re going to have to wait on this, though, because of a compatibility issue between Appium 5.0.0 and WebDriver 4.22.0.  Hopefully that will get resolved soon.

Dan

From: Troy Walsh ***@***.***>
Sent: Thursday, July 11, 2024 9:40 AM
To: OpenMAQS/.github ***@***.***>
Cc: Mcnamee, Dan ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [OpenMAQS/.github] New release? (Discussion #3)

Lots of great updates on the .Net version the last few months. Any idea on when we might see a new NuGet package? We could use some of the updates. Especially the screenshot repair for the newest Selenium versions. New version was released
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
    Report Suspicious  <https://us-phishalarm-ewt.proofpoint.com/EWT/v1/IOVPU50!fMB5MgEl5J38VqlQcOvjcJnGuE8D1zK8_9OJcsJCmTquy1hR3zFm1gwuQVTj-5aPTIKkV0gjD6zC_v29nzQMRwqiybiqkhBXbyrwzWhD5j3lbZE7jlVB3k264g$>   ‌
ZjQcmQRYFpfptBannerEnd

Lots of great updates on the .Net version the last few months. Any idea on when we might see a new NuGet package? We could use some of the updates. Especially the screenshot repair for the newest Selenium versions.

New version was released last month
https://www.nuget.org/packages/OpenMAQS.Maqs/<https://urldefense.com/v3/__https:/www.nuget.org/packages/OpenMAQS.Maqs/__;!!IOVPU50!168lJxGTTMqgd9i1BMQEZYfPEZG4PSrJ5YLf82FKSi7zoup1fhNw_g3xt33bnIeYtjWayX45LmlbS3w46HyV7g1WWg$>
